components/brave_vpn/resources/panel/assets/svg-icons/waves-dark.svg

```diff
@@ -0,0 +1,90 @@
+<svg width="320" height="104" viewBox="0 0 320 104" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This file looks like it can be reduced a bit with SVGOMG

![alt text](image.png)

Reply:
Yeah, reduced this and all other svgs also. 👍🏼